src/seed/organizationTags.json

```
@@ -0,0 +1,1733 @@
+			"ru": "Театр / Музыкальный театр / Место для перформанса",
+			"uk": "Театр / Музичний театр / Місце для перформансу"
+		},
+		"metadata": {
+
+		}
+	},
+	{
+		"type": "type.Tag",
+		"identifier": "organization.category.Theater.Entertainment",
+		"title": {
+			"de": "Entertainment",
+			"en": "Entertainment",
+			"fr": "Divertissement",
+			"es": "Entretenimiento",
+			"it": "Intrattenimento",
+			"tr": "Eğlence",
+			"ar": "الترفيه",
+			"pl": "Rozrywka",
+			"ru": "Развлечения",
+			"uk": "Розваги"
+		},
+		"metadata": {
+
+		}
+	},
+	{
+		"type": "type.Tag",
+		"identifier": "organization.category.Theater.ChildrenYouthTheater",
+		"title": {
+			"de": "Kinder- und Jugendtheater",
+			"en": "Children and Youth Theater",
+			"fr": "Théâtre pour enfants et jeunes",
+			"es": "Teatro para niños y jóvenes",
+			"it": "Teatro per bambini e giovani",
+			"tr": "Çocuk ve Gençlik Tiyatrosu",
+			"ar": "مسرح الأطفال والشباب",
+			"pl": "Teatr dziecięcy i młodzieżowy",
+			"ru": "Детский и юношеский театр",
+			"uk": "Дитячий та юнацький театр"
+		},
+		"metadata": {
+
+		}
+	},
+	{
+		"type": "type.Tag",
+		"identifier": "organization.category.Theater.Cabaret",
+		"title": {
+			"de": "Kleinkunst",
+			"en": "Cabaret",
+			"fr": "Cabaret",
+			"es": "Cabaret",
+			"it": "Cabaret",
+			"tr": "Kabare",
+			"ar": "كاباريه",
+			"pl": "Kabaret",
+			"ru": "Кабаре",
+			"uk": "Кабаре"
+		},
+		"metadata": {
+
+		}
+	},
+	{
+		"type": "type.Tag",
+		"identifier": "organization.category.Theater.Musical",
+		"title": {
+			"de": "Musical",
+			"en": "Musical",
+			"fr": "Comédie musicale",
+			"es": "Musical",
+			"it": "Musical",
+			"tr": "Müzikal",
+			"ar": "ميوزيكل",
+			"pl": "Musical",
+			"ru": "Мюзикл",
+			"uk": "Мюзикл"
+		},
+		"metadata": {
+
+		}
+	},
+	{
+		"type": "type.Tag",
+		"identifier": "organization.category.Theater.Opera",
+		"title": {
+			"de": "Oper",
+			"en": "Opera",
+			"fr": "Opéra",
+			"es": "Ópera",
+			"it": "Opera",
+			"tr": "Opera",
+			"ar": "أوبرا",
+			"pl": "Opera",
+			"ru": "Опера",
+			"uk": "Опера"
+		},
+		"metadata": {
+
+		}
+	},
+	{
+		"type": "type.Tag",
+		"identifier": "organization.category.Theater.Operetta",
+		"title": {
+			"de": "Operette",
+			"en": "Operetta",
+			"fr": "Opérette",
+			"es": "Opereta",
+			"it": "Operetta",
+			"tr": "Operet",
+			"ar": "أوبريت",
+			"pl": "Operetka",
+			"ru": "Оперетта",
+			"uk": "Оперета"
+		},
+		"metadata": {
+
+		}
+	},
+	{
+		"type": "type.Tag",
+		"identifier": "organization.category.Theater.Performance",
+		"title": {
+			"de": "Performance",
+			"en": "Performance",
+			"fr": "Performance",
+			"es": "Performance",
+			"it": "Performance",
+			"tr": "Performans",
+			"ar": "الأداء",
+			"pl": "Performance",
+			"ru": "Перформанс",
+			"uk": "Перформанс"
+		},
+		"metadata": {
+
+		}
+	},
+	{
+		"type": "type.Tag",
+		"identifier": "organization.category.Theater.PuppetFigureObjectTheater",
+		"title": {
+			"de": "Puppen- / Figuren-/ Objekttheater",
```

Hier sind die Leerzeichen zwischen den Slashes nicht konsistent. 👮 

```suggestion
			"de": "Puppen- / Figuren- / Objekttheater",
```

---

Super. Ich habe die Texte 1 zu 1 von der alten Webseite übernommen.